aztk/node_scripts/main.py

```diff
@@ -18,4 +20,5 @@ def run():
 
 
 if __name__ == "__main__":
+    logger.setup_logging()
```

Comment:
This isn't used anywhere it seems. Is this just for use later?

Reply:
There was some logging in the node script which were basically not being printed to the stdout. This I think should fix it

Reply:
Yep it works, I just updated the one still using logging which don't work